src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -590,8 +590,7 @@ public ApplicationContext createApplicationContext(LDAPSecurityRealm realm) {
         vars.putAll(getExtraEnvVars());
         initialDirContextFactory.setExtraEnvVars(vars);
 
-        FilterBasedLdapUserSearch ldapUserSearch = new FilterBasedLdapUserSearch(getUserSearchBase(), getUserSearch(), initialDirContextFactory);
```

Comment:
It doesn't look like FilterBasedLdapUserSearch is used anymore, though it is still imported and referenced several places in JavaDoc in this file.